spec/jasmine-fixture-spec.coffee

```diff
@@ -1,7 +1,9 @@
 describe "jasmine.fixture", ->
+  Then "no emmet global", -> `(1, eval)('this')`.hasOwnProperty('emmet') == false
 
   EXAMPLES = [
      'article'                                                             #<article></article>
+     'backlog-item'                                                        #<backlog-item></backlog-item>
```

Comment:
Correct. It was opened in #48 (due to angular component stuff) and i had no idea how to possibly hack that into the old zencoding port, which is what started me down this whole path